.travis.yml

```diff
@@ -17,7 +18,24 @@ before_script:
   - composer selfupdate --quiet
   - composer install --prefer-dist --no-interaction --no-progress
   - ./vendor/bin/phpunit
-  - composer update --prefer-dist --no-interaction --no-progress
 
 script:
-  - ./vendor/bin/phpunit
+  - make test
+
+after_script:
```

Comment:
Yes, only master by default (see here: https://docs.travis-ci.com/user/deployment/bintray#Branch-to-deploy-from)

I might be wrong but [`after_script` is executed **after** `deploy`](https://docs.travis-ci.com/user/customizing-the-build/#The-Build-Lifecycle), so IMHO the created phar will not be pushed to bintray.